sale_order_line_sequence/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# Copyright 2017 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Sale Order Line Sequence",
+    "summary": "Propagates SO line sequence to invoices and stock picking.",
+    "version": "11.0.1.0.0",
+    "author": "Eficent, "
+              "Serpent CS, "
+              "Odoo Community Association (OCA)",
+    "category": "Sales",
+    "website": "https://www.eficent.com/",
```

Comment:
@remihb IMO Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)